### ISSUE TITLE
General question

### ISSUE THREAD

**Issue by Kokiro** [ianc1/BaseCSS#1]
Hi,

it looks really cool!

I would like to ask some general questions:
Do you have a color palette?
How do you render the forms' field errors?
How do you do with column offsets?
Do you have some more examples I could learn from?

Thanks in advance.

**Comment by ianc1** [comment]
Hi Kokiro, 

Thank you for your feedback.

I've tried to keep the styling as minimal as possible so that project themes can be layered on top without having to reset lots of styles I've imposed. I think of Base CSS as a wire frame theme that can help you quickly build and layout your site with minimal CSS styling to allow you to focus on coding and getting the site functional. The site's unique theme can then be applied at the end using a separate CSS file which extends Base CSS.  I.e don't edit `base.css` directly so that you can easily replace the file in the future to take advantage of fixes and improvements.

* A color palette would be part of your project theme.
* A default style for displaying form errors would be a useful addition. I'll add one when I have some spare time, or if you would like to contribute feel free. This text feedback control would need to be used inside a form group.
* The grid layout is pretty basic, there isn't a concept of offsets to skip columns. You'll need to either use an empty column to provide the offset of add a left margin to your column grid.

I don't have any more examples at the moment but will add more as time allows.

Hope this helps,

Ian

**Comment by Kokiro** [comment]
Hi Ian,

thank you for taking your time for the exaustive answer.

I totally agree on the philosophy and I will just try to discuss points which cannot be adaptet on top of BaseCSS.

I have actually two:

- I suggest to rename `.form-group` in `.input-group` --> reason is `.form-group` is the class I would use for the errors.

- `.form-group > .select > select:not(:last-of-type) {
    border-right: 0;
}`
I changed the selector because if in a (for now) `.form-group` there are 2 `select`, the right one is missing the border.

Best regards,
Kokiro

**Comment by Kokiro** [comment]
Hi,

another issue is:

```
@media (min-width: 1200px) {
  .mobile-hide {
    display: block;
}
```
doesn't work well with table (maybe because I have two line in a row).
I added:

```
@media (min-width: 1200px) {
  table .mobile-hide {
    display: table-cell;
}
```

Best regards,
Kokiro

**Comment by ianc1** [comment]
Hi Kokiro,

I've added a few form improvements and some additional examples. Form groups should now support multiple select controls and there is a new `.form-feedback` class to display validation errors.

If you have any improvements/fixes that you would like me to add, just create a fork with the changes and I'll add them in.

Thanks, Ian.

**Comment by ianc1** [comment]
Hi Kokiro,

I've fixed the issue with the mobile-hide class on tables. Replicated the issue using mobile-hide on table rows or cells with wrapped text would caused misalignment of cells.

The fix is included in bug fix release 1.1.2.

Thanks, Ian
